server/command.go

```diff
@@ -181,3 +176,38 @@ func responsef(format string, args ...interface{}) *model.CommandResponse {
 		Type:         model.POST_DEFAULT,
 	}
 }
+
+func createAgendaCommand() *model.Command {
+	agenda := model.NewAutocompleteData(commandTriggerAgenda, "[command]", "Available commands: list, queue, setting, help")
+
+	list := model.NewAutocompleteData("list", "", "Show a list of items queued for the next meeting")
+	optionalListNextWeek := model.NewAutocompleteData("next-week", "(optional)", "If `next-week` is provided, it will list the agenda for the next calendar week.")
+	list.AddCommand(optionalListNextWeek)
+	agenda.AddCommand(list)
+
+	queue := model.NewAutocompleteData("queue", "", "Queue `message` as a topic on the next meeting.")
+	queue.AddStaticListArgument("If `next-week` is provided, it will queue for the meeting in the next calendar week.", false, []model.AutocompleteListItem{{
+		HelpText: "If `next-week` is provided, it will queue for the meeting in the next calendar week.",
+		Hint:     "(optional)",
+		Item:     "next-week",
+	}})
+	queue.AddTextArgument("Creates a post for user with the given message for the next meeting date.", "message", "")
```

Comment:
```suggestion
	queue.AddTextArgument("Message for the next meeting date.", "message", "")
```

Reply:
The text here is describing the meaning of the `Message` autocomplete option.  I think it makes sense to somehow tell the user `Message` is not an autocomplete value, but an input needed from the user.

On the same point, I wonder if we could/should somehow designate the difference between an autocomplete value versus an input required from the user.  As an example, `next-week` is an autocomplete value and `Message` requires free text from the user.
![image](https://user-images.githubusercontent.com/7575921/82927347-355c7400-9f46-11ea-9fa6-8bb49a88e4be.png)

Reply:
Agree. What we could do is differentiate using icons on the left side, or have different styles, but it is not yet part of the Server PR. I'll create the ticket for this.
I'd suggest using `[message]` for now.

Reply:
That'd be perfect.  Thanks!